sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -219,7 +238,22 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
       postPrepareRenderFrame()
       return
     }
-    mapboxRenderer.render()
+
+    if (widgetRenderer.hasWidgets()) {
+      if (widgetRenderer.needRender) {
```

Comment:
concat to `if (widgetRenderer.hasWidgets() && widgetRenderer.needRender)`?

Reply:
But logic below is different, wdy mean?